__tests__/workers/newView.ts

```diff
@@ -313,10 +323,12 @@ describe('reading streaks', () => {
   });
 
   it('should not increment a reading streak if lastViewAt is the same day', async () => {
-    await runTest('2024-01-26T19:17Z', '2024-01-26T17:23Z', defaultStreak, {
-      ...defaultStreak,
-      lastViewAt: new Date('2024-01-26T19:17Z'),
-    });
+    await runTest(
+      '2024-01-26T19:17Z',
+      '2024-01-26T17:23Z',
+      defaultStreak,
+      defaultStreak,
```

Comment:
`defaultStreak` does not contain the `lastViewAt` timestamp. That's why the test was sending it before. Sending just `defaultStreak` as the last parameter here means, that we do not even try to compare the dates anymore. Which again, seems to defeat the purpose of the test.

Reply:
yep, and it's okay that it does not contain it, as we are no longer comparing against the incoming date (which would require us to do timezone conversion shenanigans) 🙏

Reply:
If we do not send the `lastViewAt`, then the `expectedStreak` object does not contain the `lastViewAt` key and we do not do this comparison at all:
```typescript
if (key === 'lastViewAt') {
        expect(streak?.[key]).toEqual(lastView.timestamp);
      } else {
        expect(streak?.[key]).toEqual(expectedStreak[key]);
      }
```